subjects/devops/write_file/README.md

```diff
@@ -0,0 +1,86 @@
+## Write file
+
+### Instructions
+
+Create a file `write_file.py` which will have a function `to_do(input)` with one input a `list[tuple[dt.date, str]]` which will be a to do list example:
```

Comment:
Nothing major - my spell checker suggest to spell to do list as follows: `to-do list`

Reply:
done